src/framework/preferences/js/Panels.js

```diff
@@ -150,8 +150,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         // in order to reset the dom binder when panels are in an iframe.
         // It can be be eliminated once we have the new renderer.
         var userJQuery = that.container.constructor;
-        var oldContainer = that.container;
-        that.container = userJQuery(that.container.selector, oldContainer[0].ownerDocument);
+        var context = that.container[0].ownerDocument;
+        var selector = that.container.selector;
+        that.container = userJQuery(selector, context);
```

Comment:
See above - it should be made clear that this code is related to that within the DOM binder and that the user can go to a single place, perhaps a JIRA, in order to understand the workflow. This JIRA should link to https://issues.fluidproject.org/browse/FLUID-4536 which is our previous work in this area.